build.gradle

```diff
@@ -90,19 +92,15 @@ task itest(type: Test) {
 	useJUnitPlatform()
 }
 
-//Create "libraryJar"
-task libraryJar(type: Jar, dependsOn: classes) {
-	archiveClassifier = 'library'
-	from sourceSets.main.allSource
-}
-//Create "bootJar" already done (so "jar" task can be disabled to avoid publish conflicts)
 jar {
-	enabled = false
+	enabled = true
+	archiveClassifier.set('library')
```

Comment:
How is it different from
`archiveClassifier = 'library'`
?

Reply:
archiveClassifier is not a `String` property, it is a `Property<String>`, `archiveClassifier = 'library'` simply does not work.
Before, it was possible to write `classifier = 'library'` but this has been deprecated.